### ISSUE TITLE
Pull will delete local uncommited file changes!!!!!

### ISSUE THREAD

**Issue by Unimax** [uvicgse/project-virtual-team-1#103]
recreate Steps:

1. make a change to a file (say file.txt) locally
2. commit something to the same file and commit on the report. (using the seprate app or browsers)
3. pull in visual git

the changes made in step 1 will be lost!

**Comment by colinwerner** [comment]
@Unimax based on my testing step #2 is not required:
Steps to Reproduce:
1. edit a tracked file (observe the file is shown as shown in VG)
2. perform a 'pull' from the NavBar

Expected Results:
3. changes will be pulled (merge conflicts will be identified and may cause the pull to fail)

Actual Results:
4. the changes made in step 1 will be lost!

**Comment by hdauvic** [comment]
> @Unimax based on my testing step #2 is not required:
> Steps to Reproduce:
> 
> 1. edit a tracked file (observe the file is shown as shown in VG)
> 2. perform a 'pull' from the NavBar
> 
> Expected Results:
> 3. changes will be pulled (merge conflicts will be identified and may cause the pull to fail)
> 
> Actual Results:
> 4. the changes made in step 1 will be lost!

You are correct, I've updated the original text to be more robust and removed step 2. I also discovered that this issue persists with uncommited, but staged files.
